Jenkinsfile.conda

```diff
@@ -0,0 +1,98 @@
+pipeline {
+    agent{ 
+        docker {
+            image 'lsstts/conda_package_builder:latest'
+            alwaysPull 'true'
+            args "--env TS_CONFIG_OCS_DIR=/home/saluser/ts_config_ocs --env LSST_DDS_DOMAIN=citest -u root --entrypoint=''"
+        }
+    }
+    environment {
+        package_name = "ts_environment"
```

Comment:
Package_name is set to `ts_environment` should probably be `environment` only as the `ts-` is added already in later stages (check line 81).

Reply:
Fixed.